controllers/grid/settings/user/UserGridRow.inc.php

```diff
@@ -70,8 +71,7 @@ public function initialize($request, $template = null)
                 $actionArgs['newUserId'] = $rowId;
 
                 // Verify that the old user exists
-                $userDao = DAORegistry::getDAO('UserDAO'); /** @var UserDAO $userDao */
-                $oldUser = $userDao->getById($this->getOldUserId());
+                $oldUser = Repo::user()->get($this->getOldUserId());
```

Comment:
You've probably tested this but I'm always a little bit wary of assuming an ID is an `int` when working with the grids.

Reply:
We'll probably have a lot of cases like this to work out as we adopt strict typehinting. I've added an int cast for now.